ssm_jax/lgssm/models.py

```diff
@@ -0,0 +1,127 @@
+import jax.numpy as jnp
+from jax import lax
+import jax.random as jr
+from jax.tree_util import register_pytree_node_class
+
+from distrax import MultivariateNormalFullCovariance as MVN
+
+
+@register_pytree_node_class
+class LinearGaussianSSM:
```

Comment:
Do you want to add filter, smoother, and posterior_sample methods, that call the low-level functions?